deploy/olm-catalog/ibm-block-csi-operator/1.0.0/ibm-block-csi-operator.v1.0.0.clusterserviceversion.yaml

```diff
@@ -52,10 +52,11 @@ spec:
 
     Supported operating systems:
       - RHEL 7.x (x86 architecture)
-      
+
+    > **Note**: The operator and its custom resource must be installed in the kube-system namespace.
```

Comment:
1. I think you don't need to mention the custom resource here, because they are always in the same project, when you switch to another project, you can not see the operator at all, so it's impossible to create it in another project.
2. `kube-system`, add "`"
3. use project, not namespace

Please ask Rivka to review it

Reply:
done